cromshell

```diff
@@ -730,11 +753,79 @@ function execution-status-count()
     return 11
   fi
 
-  # Make it pretty for the user:
-  # {tasks:{status: count}}
-  jq '.calls | map_values(group_by(.executionStatus) | map({(.[0].executionStatus): . | length}) | add)' "$f"
+  if ${doPretty}; then
```

Comment:
Can you also update the README with these options for `execution-status-count`?

Reply:
Updated!